helm/deployments/openshift/tekton/build-pipeline.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: build-pipeline
+spec:
+  params:
+    - description: The name of the secret holding the Rocket.Chat URL
+      name: rocketchat-url-secret
+      type: string
+  tasks:
+    - name: start-build
+      taskRef:
+        kind: Task
+        name: start-build
+    - name: send-to-rocketchat
+      params:
+        - name: url-secret
+          value: $(params.rocketchat-url-secret)
```

Comment:
is this a helm value? and if so i suspect there is a follow up pr for the tenant repo!

Reply:
No, it is the name of the openshift secret that the curl task (see that file below) will look up and use. It is provided by the pipeline-run (in this case "rocketchat-url").